e2e-test/e2e/chaos/dnschaos/dns.go

```diff
@@ -80,7 +80,7 @@ func TestcaseDNSRandom(
 	framework.ExpectNoError(err, "create dns chaos error")
 
 	for _, domainName := range effectDomainNames {
-		err = wait.Poll(time.Second, 5*time.Second, func() (done bool, err error) {
+		err = wait.Poll(time.Second, 10*time.Second, func() (done bool, err error) {
```

Comment:
I remember this issue happened in recent days. Why does this issue not happen before?